packages/components/src/card/src/Card.css

```diff
@@ -1,33 +1,32 @@
 .o-ui-card {
     display: grid;
-    background-color: var(--o-ui-bg-alias-surface);
+    border-radius: var(--hop-shape-rounded-md);
 }
 
 /* VARIANT */
-.o-ui-card-elevated {
-    box-shadow: var(--o-ui-bs-alias-lifted);
-    border-radius: var(--o-ui-br-4);
+.o-ui-card-outline {
+    background-color: var(--hop-neutral-surface);
+    border: 1px solid var(--hop-neutral-border-weak);
 }
 
-.o-ui-card-outline {
-    border: 1px solid var(--o-ui-b-alias-low-break);
-    border-radius: var(--o-ui-br-3);
+.o-ui-card-elevated {
+    background-color: var(--hop-neutral-surface-weakest);
 }
 
 /* SIZES */
 .o-ui-card-xs {
     width: 100%;
-    max-width: var(--o-ui-sz-15);
+    max-width: 16rem;
```

Comment:
If we rely on the Figma, there is no notion of size at the card. How should we approach this problem?

Reply:
The idea with the rebrand is to apply colors, etc and not change behaviours ideally.